source/features/mark-merge-commits-in-list.tsx

```diff
@@ -27,7 +27,7 @@ const filterMergeCommits = async (commits: string[]): Promise<string[]> => {
 
 	const mergeCommits = [];
 	for (const [key, commit] of objectEntries(repository)) {
-		if (commit.parents.totalCount >= 2) {
+		if (commit?.parents.totalCount >= 2) {
```

Comment:
here is to fix below.

![CleanShot 2024-08-14 at 11 26 02@2x](https://github.com/user-attachments/assets/1bc65c0b-09a2-4589-9297-e9591586a08e)